interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1175 @@
+{
```

Comment:
<p>I think we talked about the "quick and dirty" trick of </p><pre class="ql-syntax" spellcheck="false">conda run -n tess pip install [package]
</pre><p>This method won't persist, but it's a good way to get a package running quickly. We can also link to the "real" methods of handling packages: <a href="https://outerspace.stsci.edu/display/MASTDOCS/Managing+Software" target="_blank">Managing+Software</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>